examples/org.eclipse.swt.snippets/src/org/eclipse/swt/snippets/Snippet384.java

```diff
@@ -0,0 +1,418 @@
+/*******************************************************************************
+ * Copyright (c) 2025 Vector Informatik GmbH and others.
+ *
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ *******************************************************************************/
+package org.eclipse.swt.snippets;
+
+import java.io.*;
+import java.util.*;
+import java.util.List;
+import java.util.function.*;
+
+import org.eclipse.swt.*;
+import org.eclipse.swt.graphics.*;
+import org.eclipse.swt.layout.*;
+import org.eclipse.swt.widgets.*;
+
+public class Snippet384 {
```

Comment:
Please add a comment to the class used as the description of the snippet shown in the snippet explorer (see other snippets for a template).